manim/config.py

```diff
@@ -81,346 +79,19 @@ def _parse_config(config_parser, args):
-    parser.add_argument(
-        "-r", "--resolution",
-        help="Resolution, passed as \"height,width\"",
-    )
-
-    # This sets FROM_ANIMATION_NUMBER and UPTO_ANIMATION_NUMBER
-    parser.add_argument(
-        "-n", "--from_animation_number",
-        help="Start rendering not from the first animation, but"
-             "from another, specified by its index.  If you pass"
-             "in two comma separated values, e.g. \"3,6\", it will end"
-             "the rendering at the second value",
-    )
-
-    # Specify the manim.cfg file
-    parser.add_argument(
-        "--config_file",
-        help="Specify the configuration file",
-    )
-
-    return parser.parse_args(arg_list)
-
-
-def _init_dirs(config):
-    # Make sure all folders exist
-    for folder in [config["media_dir"], config["video_dir"],
-                   config["tex_dir"], config["text_dir"]]:
-        if not os.path.exists(folder):
-            os.makedirs(folder)
-
-
-def _from_command_line():
-    """Determine if manim was called from the command line."""
-    # Manim can be called from the command line in three different
-    # ways.  The first two involve using the manim or manimcm commands
-    prog = os.path.split(sys.argv[0])[-1]
-    from_cli_command = prog in ['manim', 'manimcm']
-
-    # The third way involves using `python -m manim ...`.  In this
-    # case, the CLI arguments passed to manim do not include 'manim',
-    # 'manimcm', or even 'python'.  However, the -m flag will always
-    # be the first argument.
-    from_python_m = sys.argv[0] == '-m'
-
-    return from_cli_command or from_python_m
 
+    return config
 
-# Config files to be parsed, in ascending priority
-library_wide = os.path.join(os.path.dirname(__file__), 'default.cfg')
-config_files = [
-    library_wide,
-    os.path.expanduser('~/.manim.cfg'),
-]
 
+args, config_parser, file_writer_config, successfully_read_files = _run_config()
 if _from_command_line():
-    args = _parse_cli(sys.argv[1:])
-    if args.config_file is not None:
-        if os.path.exists(args.config_file):
-            config_files.append(args.config_file)
-        else:
-            raise FileNotFoundError(f"Config file {args.config_file} doesn't exist")
-    else:
-        script_directory_file_config = os.path.join(os.path.dirname(args.file), 'manim.cfg')
-        if os.path.exists(script_directory_file_config):
-            config_files.append(script_directory_file_config)
-
-else:
-    # In this case, we still need an empty args object.
-    args = _parse_cli([], input=False)
-    # Need to populate the options left out
-    args.file, args.scene_names, args.output_file = '', '', ''
-
-config_parser = configparser.ConfigParser()
-successfully_read_files = config_parser.read(config_files)
-logger.info(f'Read configuration files: {successfully_read_files}')
-
-# this is for internal use when writing output files
-file_writer_config = _parse_file_writer_config(config_parser, args)
```

Comment:
question: why was this removed?

Reply:
Do see line [92](https://github.com/ManimCommunity/manim/blob/561f9bdb3ff095ec48f449b9c058f12640b9d7b5/manim/config.py#L92). I defined it there.

Reply:
You're right!